src/sdk/namespace_nb.js

```diff
@@ -86,9 +83,6 @@ class NamespaceNB {
             client: object_sdk.rpc_client,
             bucket: this.target_bucket,
         }, params);
-        // Noobaa bucket does not currrently support partNumber query parameter. Ignore it for now.
-        // If set, part_number is positive integer from 1 to 10000
-        if (params.part_number) _.unset(params, 'part_number');
```

Comment:
as a follow-up to the comment above in `s3_get_object`, maybe you should always pass `params.part_number` when it's set on the S3 request, and just leave this `_.unset` here

Reply:
So guess we agreed this looks good.